app/contract-boost-calculator/page.tsx

```diff
@@ -83,9 +100,17 @@ export default function ContractBoostCalculator() {
     }));
   }, [equippedArtifactsListForSelectedIGN]);
```

Comment:
@DukeCephalopod idk enough about react to know if `getEquippedArtifactById` needs to be added here. eslint is saying it has to

Reply:
eeurgh, mixed hooks and instance functions. It should be added here, yes, altho it won't matter in the architecture as written.